<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_TaskZo</class>
 <widget class="QDialog" name="Dialog_TaskZo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>302</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Зона ограничения</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton_Cancel">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_Ok">
     <property name="text">
      <string>Ок</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Режим расчета:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButton_Graphics">
        <property name="text">
         <string>Построение границ зоны</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="radioButton_Text">
        <property name="text">
         <string>Таблично текстовый документ</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Высоты точек наблюдения, м:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="1">
       <widget class="DotLineEdit" name="lineEdit_Height"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>220</height>
      </size>
     </property>
     <property name="title">
      <string>Область расчета:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>dY,м:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Ymax,м:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Ymin,м:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Xmax,м:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>dX,м:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Xmin,м:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="DotLineEdit" name="lineEdit_Xd"/>
      </item>
      <item row="1" column="0">
       <widget class="DotLineEdit" name="lineEdit_Xmin"/>
      </item>
      <item row="1" column="1">
       <widget class="DotLineEdit" name="lineEdit_Xmax"/>
      </item>
      <item row="3" column="0">
       <widget class="DotLineEdit" name="lineEdit_Ymin"/>
      </item>
      <item row="3" column="1">
       <widget class="DotLineEdit" name="lineEdit_Ymax"/>
      </item>
      <item row="3" column="2">
       <widget class="DotLineEdit" name="lineEdit_Yd"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DotLineEdit</class>
   <extends>QLineEdit</extends>
   <header>../Widget/DotLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
